tests/integration.rs

```diff
@@ -6772,4 +6772,216 @@ mod tests {
+                    String::from_utf8_lossy(&output.stdout), String::from_utf8_lossy(&output.stderr));
+                }
+                // The 'receive-migration' command should be executed successfully within the given timeout
+                if let Some(status) = receive_migration
+                    .wait_timeout(std::time::Duration::from_secs(30))
+                    .unwrap()
+                {
+                    success = status.success();
+                } else {
+                    success = false;
+                }
+                if !success {
+                    let _ = receive_migration.kill();
+                    let output = receive_migration.wait_with_output().unwrap();
+                    eprintln!("\n\n==== Start 'receive_migration' output ====\n\n---stdout---\n{}\n\n---stderr---\n{}\n\n==== End 'receive_migration' output ====\n\n",
+                    String::from_utf8_lossy(&output.stdout), String::from_utf8_lossy(&output.stderr));
+                }
+                assert!(
+                    success,
+                    "Unsuccessful command: 'send-migration' or 'receive-migration'."
+                );
+            });
+
+            let print_and_panic = |src_vm: Child, dest_vm: Child, message: &str| -> ! {
+                let mut src_vm = src_vm;
+                let mut dest_vm = dest_vm;
+
+                let _ = src_vm.kill();
+                let src_output = src_vm.wait_with_output().unwrap();
+                eprintln!(
+                    "\n\n==== Start 'source_vm' stdout ====\n\n{}\n\n==== End 'source_vm' stdout ====",
+                    String::from_utf8_lossy(&src_output.stdout)
+                );
+                eprintln!(
+                    "\n\n==== Start 'source_vm' stderr ====\n\n{}\n\n==== End 'source_vm' stderr ====",
+                    String::from_utf8_lossy(&src_output.stderr)
+                );
+                let _ = dest_vm.kill();
+                let dest_output = dest_vm.wait_with_output().unwrap();
+                eprintln!(
+                    "\n\n==== Start 'destination_vm' stdout ====\n\n{}\n\n==== End 'destination_vm' stdout ====",
+                    String::from_utf8_lossy(&dest_output.stdout)
+                );
+                eprintln!(
+                    "\n\n==== Start 'destination_vm' stderr ====\n\n{}\n\n==== End 'destination_vm' stderr ====",
+                    String::from_utf8_lossy(&dest_output.stderr)
+                );
+
+                panic!("Test failed: {}", message)
+            };
+
+            // Check and report any errors occured during the live-migration
+            if r.is_err() {
+                print_and_panic(src_child, dest_child, "Error occured during live-migration");
+            }
+
+            // Check the source vm has been terminated successful (give it '1s' to settle)
+            thread::sleep(std::time::Duration::new(1, 0));
```

Comment:
Can we extend the time here to `3s`? In my local test on arm64 machines, 1s seems to be not enough and sometimes it will give:
```
thread 'main' panicked at 'Test failed: source VM was not terminated successfully.', tests/integration.rs:7001:17
```

I tried to extend here to 3s and it will pass on arm.
```
running 1 test
test tests::live_migration::test_live_migration ... ok
```

Reply:
Cool. This is a good finding. I will update the PR accordingly.

Reply:
@MrXinWang In addition to what you suggested above, I also made a few more changes: 

1. Removed the "x86_64" feature gate for the "live-migration" test mod, so that we can run it on the aarch64 worker once your PR is landed; 

2. Added a "1s" sleep between the "receive-migration" and "send-migration", which should prevent the "random" failures of `VmSendMigration(VmSendMigration(MigrateSend(Error connecting to UNIX socket: No such file or directory (os error 2))))`.

Reply:
Perfect! Great thanks @likebreath! I will do the refactor that @sboeuf  suggested in https://github.com/cloud-hypervisor/cloud-hypervisor/pull/3049 and take care of the rest on arm64 :))

Reply:
Yes, I am also following that PR and agrees with the proposal. Thanks again for your hard work and prompt response.